pootle/checks.py

```diff
@@ -0,0 +1,158 @@
+from django.core import checks
+from django.utils.translation import ugettext as _
+
+
+# Minimum Translate Toolkit version required for Pootle to run.
+TTK_MINIMUM_REQUIRED_VERSION = (1, 11, 0)
+
+# Minimum Django version required for Pootle to run.
+DJANGO_MINIMUM_REQUIRED_VERSION = (1, 7, 5)
+
+# Minimum lxml version required for Pootle to run.
+LXML_MINIMUM_REQUIRED_VERSION = (2, 2, 2, 0)
+
+# Minimum Redis server version required.
+# Initially set to some minimums based on:
+# 1. Ubuntu 12.04LTS's version 2.8.4 (10.04LTS was too old for RQ)
+# 2. RQ requires >= 2.6.0, and
+# 3. Wanting to insist on at least the latest stable that devs are using i.e.
+#    2.8.* versions of Redis
```

Comment:
I do, so that nobody has to figure out why its an arbitrary number, because it is arbitrary without that justification.  At some future date it might not be needed.